src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.targets

```diff
@@ -304,7 +304,7 @@ Copyright (c) .NET Foundation. All rights reserved.
       </PropertyGroup>
     </When>
     
-    <When Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework' and '$(OutputType)' == 'Exe'">
+    <When Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework' and '$(IsExecutable)' == 'true'">
```

Comment:
@eerhardt 

> I was under the assumption that things like Test projects were going to set $(IsExecutable) to true. 

Yes, thats true.

>  If that's the case, then I don't think these two lines should be changed.

We still need it so the Windows Application output type projects can still be run

> I think we should think about the best name to use for this property.

I am open for a better name. But I feel `IsExecutable` conveys its purpose to the point.